### PR TITLE
Support g2o viewer build on windows

### PR DESCRIPTION
**Pull request by PTavse**

### CODE REVIEW

**Comment by paul-michalik** [comment]
Why reopen an already merged pull request? Please create a new one from a branch created for the purpose of the task.

OK, I apologize, my mistake. This is a new branch the name just sounds similar... What confuses me is that all the commits which were already merged on master are now part of this PR. From which commit have you branched?

File: scripts/windows/bootstrap.bat [hunk 1]
```diff
@@ -55,3 +55,27 @@ rem ==============================
 set "VcPkgDeps=eigen3 suitesparse clapack openblas ceres qt5-base"
 call "%VcPkgDir%\vcpkg.exe" upgrade %VcPkgDeps% --no-dry-run --triplet %VcPkgTriplet%
 call "%VcPkgDir%\vcpkg.exe" install %VcPkgDeps% --triplet %VcPkgTriplet%
+
+
+rem ==============================
+rem Download and build QGLViewer.
+rem ==============================
```
Comment (paul-michalik) [inline scripts/windows/bootstrap.bat]:
```
rem 
rem ... if it does not exist already...?
rem 
set "QGLViewerPath=%~dp0..\..\Thirdparty\libQGLViewer-2.7.1\"
set "QGLVIEWERROOT=%QGLViewerPath%\QGLViewer" 
if not exists "%QGLVIEWERROOT%\QGLViewer2.dll" (
    rem all the statements below...
)
```

Comment (paul-michalik) [reply scripts/windows/bootstrap.bat]: Maybe even factor out to a separate script e.g. `bootstrap-ogl-viewer.bat` and invoke 
```
call "%~dp0bootstrap-ogl-viewer.bat"
```

**Review by paul-michalik — CHANGES_REQUESTED** [review]

**Comment by PTavse** [comment]
I guess i branched the code before these pull requests were merged. If I am not wrong i would have to rebase the branch? 
About the QGLViewer build, there could be two ways to provide the libraries:
1) Download and build the code while building g2o code as i did in the first commit.
2) We can just directly make the libraries available in "Thirdparty" folder and just set QGLVIEWERROOT to the path to Thirdparty folder, as ansuman suggested.
Which one would you suggest?

**Comment by paul-michalik** [comment]
No, you have branched several days after the pull request were merged. What I don't understand is where you have branched from. You should have branched from master at `24db3a771df48fe26192e433504f4829597b97f2` but I don't see that commit at all. Strange, we need to get rid of this.

I'd prefer 1. possibility. Only this guarantees consistency across all dependencies...

**Pull request closed without merging**